packages/wrangler/src/dev/start-server.ts

```diff
@@ -345,15 +345,6 @@ export async function startLocalServer({
 			);
 		}
 
-		// In local mode, we want to copy all referenced modules into
```

Comment:
following up the call chain, this ultimately gets called in `unstable_dev()` afaict. Is `/fixtures/local-mode-tests` a good place to test this?

Reply:
Perhaps in `fixtures/local-mode-tests/tests/module.test.ts`?
This is a local-mode test that has external modules.

Reply:
We should be able to remove it from here too, yes?

https://github.com/cloudflare/wrangler2/blob/main/packages/wrangler/src/dev/local.tsx#L168-L175